math-scala/src/main/scala/org/apache/mahout/math/drm/package.scala

```diff
@@ -80,9 +80,6 @@ package object drm {
   /** Just throw all engine operations into context as well. */
   implicit def ctx2engine(ctx: DistributedContext): DistributedEngine = ctx.engine
 
-  implicit def drm2drmCpOps[K: ClassTag](drm: CheckpointedDrm[K]): CheckpointedOps[K] =
```

Comment:
it already exits here: https://github.com/apache/mahout/blob/master/math-scala/src/main/scala/org/apache/mahout/math/drm/RLikeDrmOps.scala#L137. if you upgrade to 2.11 this would crash. and both are imported into scope. Why this does not crash on 2.10 i have no idea.
Not sure which one that should be removed, but I chose this one.